### DISCUSSION TITLE
Release for Linux aarch64

### DISCUSSION THREAD

**Discussion by shiumano** [ppy/osu#25509] [Ideas]
*This is a re-creation of https://github.com/ppy/osu/discussions/14537.*

It has already been two years since this discussion ended.

osu!lazer can already be played on Linux aarch64.

**Comment by peppy** [comment]
Your best bet is +1ing the other thread to show your support. We'd either need to see enough users of this architecture/platform to sink the effort, or have someone else take on the change.

**Reply by lokxii** [reply]
I have been running osu!lazer playing osu standard for a while by directly running `dotent run`. Everything seems fine. I'm on asahi linux. But since it is not an official release I can't submit scores.

Actually I wonder what is  missing / preventing an official release for Linux aarch64?
